src/components/redux/actions/actionsTypes.js

```diff
@@ -1 +1,10 @@
-export const FIRST_TYPE = "CURRENT_BOARD_ID";
+export const ADD_TO_CART = "ADD_TO_CART";
+export const DELETE_FROM_CART = "DELETE_FROM_CART";
+export const PRODUCT_QUANTITY = "PRODUCT_QUANTITY";
+export const MODIFY_QUANTITY = "MODIFY_QUANTITY";
+export const QUANTITY_IN_SINGLE_PRODUCT = "QUANTITY_IN_SINGLE_PRODUCT";
+export const SUM_TOTAL_PRICE = "SUM_TOTAL_PRICE";
```

Comment:
actions are not used to compute things, actions are used to change the state. so here summing the total price is more a computation as a result of state, if that makes sense, not something that you're doing to change the state